packages/dds/task-manager/src/test/taskManager.spec.ts

```diff
@@ -30,23 +30,28 @@ function createConnectedTaskManager(id: string, runtimeFactory: MockContainerRun
 }
 
 function createDetachedTaskManager(id: string, runtimeFactory: MockContainerRuntimeFactory):
-    { taskManager: TaskManager; attach: () => void; } {
+    { taskManager: TaskManager; attach: () => Promise<void>; } {
     // Create a detached TaskManager.
     const dataStoreRuntime = new MockFluidDataStoreRuntime();
     const containerRuntime = runtimeFactory.createContainerRuntime(dataStoreRuntime);
     dataStoreRuntime.local = true;
     const clientId = dataStoreRuntime.clientId;
 
     const taskManager = new TaskManager(id, dataStoreRuntime, TaskManagerFactory.Attributes);
-    const attach = () => {
+    const attach = async () => {
         const services = {
             deltaConnection: containerRuntime.createDeltaConnection(),
             objectStorage: new MockStorage(),
         };
         dataStoreRuntime.local = false;
         taskManager.connect(services);
+
+        // Manually trigger a summarize (should be done automatically when attaching normally)
+        await taskManager.summarize();
```

Comment:
Probably move this up above the local/connect calls since in normal attach the summarization should happen first